docker_daemon/metadata.csv

```diff
@@ -1,4 +1,5 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+docker.cpu.limit,gauge,,percent,,"Limit on CPU available to the container, expressed as percentage of a core",0,docker,docker.cpu.limit
```

Comment:
`docker.cpu.limit` is reported in `UserHz` (1/100s), not in percentage, which makes me other metrics in this doc are not accurately documented either.

Reply:
It's true that it's not technically a percentage, but is there a meaningful difference? If the container is limited to 1 cpu core, then the value would be 100. If it's half a core it's 50 and if it's two cores it's 200. I could change it to `UserHz`, but no one knows what that is. :P